lib/ansible/modules/system/aix_filesystem.py

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: Kairo Araujo (@kairoaraujo)
```

Comment:
`author` is a list, so it should be:
```yaml
author:
- Kairo Araujo (@kairoaraujo)
```

Reply:
We are moving to the use of lists, because changes to the author list only affects the new line. The same reason we prefer trailing comma's in multi-line lists or dicts in python.

For new modules it's better to start off with lists.

Reply:
1) There is rarely more than one author.
2) The same principle is used elsewhere in Ansible that you can use either string or list.